registration/registration-client/src/main/java/io/mosip/registration/util/control/impl/DocumentFxControl.java

```diff
@@ -65,13 +66,16 @@ public class DocumentFxControl extends FxControl {
 	private String PREVIEW_ICON = "previewIcon";
 
 	private String CLEAR_ID = "clear";
+	
+	private GenericController genericController;

```

Comment:
Can we only make the "notification label" available in DocumentFxControl, instead of setting the GenericController?